### DISCUSSION TITLE
Request for Billing Resolution and Activation of Benefits

### DISCUSSION THREAD

**Discussion by FabianaCampanari** [community/community#134864] [GitHub Education]
### Hi everyone,

Hello,

I received the email informing me about a billing issue with my account. I would like to clarify that there was a charge, but I renewed my GitHub Pro subscription with documentation proving that I am currently enrolled in university, which exempts me from this charge.

I was approved for the renewal of the GitHub Pro package on Sunday, August 4, 2024, but the resources have not yet been activated. Could you please let me know how I should proceed to have the approved benefits take effect?

Additionally, I would like to understand how the billing issue will be handled, which occurred because my previous benefit expired and was not accounted for. As soon as I became aware of the need for renewal, I promptly completed it, but I have yet to see the benefits reflected in my account.

Thank you for your attention, and I look forward to your guidance on how to resolve my situation.

Best regards,  
Fabiana Campanari

**Comment by eliancheng** [comment]
Hi Fabiana,

I recommend contacting GitHub Support directly through their help page. Provide them with the details of your situation, including the documentation proving your university enrollment and the date you renewed your GitHub Pro subscription.

Explain that you were approved for the renewal but have not yet seen the benefits activated in your account. They will be able to clarify how the billing issue will be handled and assist you in ensuring your benefits take effect.

**Reply by FabianaCampanari** [reply]
I just did that ! 

Thank you
